src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -123,25 +83,40 @@ public static void ValidateDA(string value, string name)
                 return;
             }
 
-            if (!DateTime.TryParseExact(value, DateFormatDA, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal, out _))
+            if (!TryParseDA(value, out _))
             {
                 throw new DicomElementValidationException(name, value, DicomVR.DA, DicomCoreResource.ValueIsInvalidDate);
             }
         }
 
+        internal static bool TryParseDA(string value, out DateTime dateTime)
+        {
+            return DateTime.TryParseExact(value, DateFormatDA, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal, out dateTime);
+        }
+
         internal static void ValidateDS(string value, string name)
         {
             ValidateLength(value.Length, 0, 16, DicomVR.DS, name, value);
         }
 
         internal static void ValidateDT(string value, string name)
```

Comment:
why are some internal? and some methods public here?

This class should be purely used to expose validate only

Reply:
I found these method don't need to be public, and haven't' change old code. I'll try to change all to internal

Reply:
I can make the tryParseXX method private, but I have a question:
If the purpose of MinimumValidation is to make sure store works, why not use same set of Parse logic?

Reply:
Parse could be different depending on the data store right? Like for SQL we convert it into a specific preformat, while validation is to just check if the string is a valid date